css/searchForm.css

```diff
@@ -63,10 +63,12 @@
   font-weight: bold;
 }
 
-.advanced-search-fields {
-  display: grid;
-  grid-template-columns: [col] repeat(3, calc(100% / 3 - 10px));
-  width: 100%;
+@media (min-width: 480px) {
+  .advanced-search-fields {
+    display: grid;
+    grid-template-columns: [col] repeat(3, calc(100% / 3 - 10px));
```

Comment:
If I understand what this is trying to do right, I think:
```
grid-template-columns: repeat(3, 1fr);
gap: 10px;
```
might be more standard/speedy?

There's probably an even more modern/responsive approach using `clamp` instead of the media query, but swapping media queries for clamp across civi could be it's own issue/project.

Reply:
But otherwise all looks good to me - great to get this screen responsive.

Reply:
Interesting - the grid CSS was already there, I only added a media query, and admittedly copied the first relevant media query I found in the CSS (hence 480px, which is small, but it seems to be OK).

I updated the property to avoid using `calc`. The `[col]` seems necessary for some span elements later on. Thanks!

Reply:
Ah yes I thought I recognised that!